src/components/LHNOptionsList/LHNOptionsList.js

```diff
@@ -62,13 +142,36 @@ function LHNOptionsList({style, contentContainerStyles, data, onSelectRow, optio
      *
      * @return {Component}
      */
-    const renderItem = ({item}) => (
-        <OptionRowLHNDataWithFocus
-            reportID={item}
-            viewMode={optionMode}
-            shouldDisableFocusOptions={shouldDisableFocusOptions}
-            onSelectRow={onSelectRow}
-        />
+    const renderItem = useCallback(
+        ({item}) => {
```

Comment:
Please rename `item` to `reportID` so that it's clear what it is.

Reply:
done ✅